### PR TITLE
Version/rescue

### PR DESCRIPTION
**Pull request by mejango**

### CODE REVIEW

File: contracts/TerminalV1Rescue.sol [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    // Get a reference to the current funding cycle for the project.
+    FundingCycle memory _fundingCycle = fundingCycles.get(fundingCycles.latestIdOf(_projectId));
+
+    // Only one-time funding cycles are rescuable.
+    require(_fundingCycle.discountRate == 201, 'cant rescue');
+
+    // Send funds to beneficiary.
+    if (_originalBalance > 0) Address.sendValue(_beneficiary, _amount);
+
+    balanceOf[_projectId] = balanceOf[_projectId] - _amount;
```
Comment (simon-something) [inline contracts/TerminalV1Rescue.sol:143]: @mejango isn't there a reentry vulnerability here (a bad projectOwner draining the terminal)? Would swap ln144 and 146

**Comment by simon-something** [comment]
Pingy ping, github, plz, work with us

File: contracts/TerminalV1Rescue.sol (hunk 1 above)
Comment (mejango) [reply contracts/TerminalV1Rescue.sol:143]: great great great eye

File: contracts/TerminalV1Rescue.sol [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+    emit PrintTickets(_projectId, _beneficiary, _amount, _memo, msg.sender);
+  }
+
+  /** 
+    @notice
+    A function that burns the supply of the dead address for a project. 
+
+    @dev
+    Callable by anyone.
```
Comment (simon-something) [inline contracts/TerminalV1Rescue.sol]: only by projectOwner ?

**Pull request merged**